host

```diff
@@ -3,11 +3,14 @@ pairinguser="pairing@mumble.ugent.be"
 
 set -e
 
-abduco   > /dev/null 2>&1 || { echo 'abduco found: no' && exit 1; }
-socat -h > /dev/null 2>&1 || { echo 'socat found: no' && exit 1; }
-ssh -V   > /dev/null 2>&1 || { echo 'ssh found: no' && exit 1; }
-uuidgen  > /dev/null 2>&1 || { echo 'uuidgen found: no' && exit 1; }
-dvtm -v  > /dev/null 2>&1 || { echo 'dvtm found: no' && exit 1; }
+for COMMAND in abduco socat ssh uuidgen dvtm
+do
+	if ! command -v $COMMAND &> /dev/null
```

Comment:
`&>` works only in bash, not all POSIX sh's. Could you replace this with `> /dev/null 2>&1` again?